awx/main/scheduler/task_manager.py

```diff
@@ -626,15 +625,21 @@ def process_pending_tasks(self, pending_tasks):
 
             # All task.capacity_type == 'control' jobs should run on control plane, no need to loop over instance groups
             if task.capacity_type == 'control':
+                if not self.instance_groups[self.controlplane_ig.name].has_remaining_capacity(control_impact=True):
+                    logger.debug(f"Skipping instance group {self.controlplane_ig.name}, already met max forks allowed to execute on Instance Group")
```

Comment:
This log statement could use some added context about `task`, otherwise it'll be a bunch of repeated and indecipherable logs.